charts/k8s/values.yaml

```diff
@@ -138,7 +138,7 @@ syncer:
 
 # Etcd settings
 etcd:
-  image: k8s.gcr.io/etcd:3.5.1-0
+  image: k8s.gcr.io/etcd:3.5.4-0
```

Comment:
FYI, this is based on https://github.com/kubernetes/kubernetes/blob/v1.25.0/build/dependencies.yaml#L57-L76